numpy/random/tests/test_random.py

```diff
@@ -515,7 +515,12 @@ def test_beta(self):
 
     def test_binomial(self):
         np.random.seed(self.seed)
-        actual = np.random.binomial(100.123, .456, size=(3, 2))
+        if sys.version_info[:2] < (3, 8):
+            actual = np.random.binomial(100.123, .456, size=(3, 2))
```

Comment:
Here and in the next one, is there a reason to keep the test for the float to int coercion? I'd just replace it.

Reply:
Yeah, was pretty 50/50 on that. So changed it now, the only point would be that we do not accidentally start deprecating or removing it when python is already doing it for us slowly ;).